easybuild/easyconfigs/r/RELION/RELION-2.0.1-intel-2016b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.0.1'
+commit_id = 'a6225608e270'
```

Comment:
Well, its a bio software, so I did not find how could we get version info. As a matter of fact, I have searched all the sources for 2.0.1, and I did not find anything...